pkg/controller/networkpolicy/antreanetworkpolicy.go

```diff
@@ -86,6 +86,11 @@ func (n *NetworkPolicyController) processAntreaNetworkPolicy(np *crdv1alpha1.Net
 	appliedToGroups := map[string]*antreatypes.AppliedToGroup{}
 	addressGroups := map[string]*antreatypes.AddressGroup{}
 	rules := make([]controlplane.NetworkPolicyRule, 0, len(np.Spec.Ingress)+len(np.Spec.Egress))
+	// clusterSetScopeSelectors keeps track of all the ClusterSet-scoped selectors of the policy.
+	// During policy peer processing, any ClusterSet-scoped selector will be registered with the
+	// labelIdentityInterface and added to this list. By the end of the function, this list will
+	// be used to remove any stale selector from the policy in the labelIdentityInterface.
+	var clusterSetScopeSelectors []*antreatypes.GroupSelector
```

Comment:
It seems it only needs the selectorKey, and it could make `RemoveStalePolicySelectors` simpler if we just collect keys here?

Reply:
Done